src/api.rs

```diff
@@ -274,3 +289,36 @@ impl NameComDnsApi {
             .collect())
     }
 }
+
+#[tokio::test]
+async fn test_namecom_api_proxy_feature() {
+    env::set_var(ENV_NAMECOM_REQUEST_PROXY, "socks5://localhost:7890");
+
+    let api = NameComDnsApi::create("", "", "https://api.name.com", 0).unwrap();
+    let result = api
+        .client
+        .get("http://www.google.com/generate_204")
```

Comment:
For this test, is the plan to also open a proxy server inside the test code?

In addition, is there a way to tell if the proxy was actually used? I think it might work if the proxy adds a header, and then we do `GET https://httpbin.org/headers` here, and verify if the proxy header was addd.